src/test/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/caseofficer/CaseOfficerRespondentEvidenceSubmittedPersonalisationTest.java

```diff
@@ -123,11 +138,22 @@ public void should_return_personalisation_when_all_mandatory_information_given()
                 .put("appellantGivenNames", "")
                 .put("appellantFamilyName", "")
                 .put("linkToOnlineService", iaExUiFrontendUrl)
+                .put("legalRepCompany", "")
+                .put("legalRepCompanyAddress", "")
+                .put("legalRepName", "")
+                .put("legalRepEmail", "")
+                .put("legalRepReference", "")
                 .build();
 
         when(asylumCase.read(APPEAL_REFERENCE_NUMBER, String.class)).thenReturn(Optional.empty());
         when(asylumCase.read(APPELLANT_GIVEN_NAMES, String.class)).thenReturn(Optional.empty());
         when(asylumCase.read(APPELLANT_FAMILY_NAME, String.class)).thenReturn(Optional.empty());
+        when(asylumCase.read(LEGAL_REP_COMPANY, String.class)).thenReturn(Optional.empty());
+        when(asylumCase.read(LEGAL_REP_COMPANY_ADDRESS, String.class)).thenReturn(Optional.empty());
+        when(asylumCase.read(LEGAL_REP_NAME, String.class)).thenReturn(Optional.empty());
+        when(asylumCase.read(LEGAL_REPRESENTATIVE_EMAIL_ADDRESS, String.class)).thenReturn(Optional.empty());
+        when(asylumCase.read(LEGAL_REP_REFERENCE_NUMBER, String.class)).thenReturn(Optional.empty());
+
 
         Map<String, String> actualPersonalisation =
             caseOfficerRespondentEvidenceSubmittedPersonalisation.getPersonalisation(asylumCase);
```

Comment:
Do you not need some tests for the formatCompanyAddress() method?

Reply:
I have created a test class AddressFormatterTest.jave for all test cases related to this formatAddress().